_topics/en/vaults.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Vaults
+
+## Optional.  An entry will be added to the topics index for each alias
+#aliases:
+#  - Foo
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Scripts and Addresses
+  - Security Enhancements
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Vaults** are a type of covenant that requires two separate
+  transactions to appear in two different blocks in order for a user to
+  spend money from their wallet.  The first transaction signals that
+  someone is attempting to spend the money and gives the user a chance
+  to block the second transaction that completes the spend.
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  Usually, there's a minimum amount of time or number of blocks that
```

Comment:
There seems to be some difference between the excerpt "transactions ... appear in two different blocks" and this "Usually, there's a minimum amount of time or blocks".

The first states that the transactions must appear in different blocks. The second implies with the "usually" that there could be cases where the transactions appear in the same block.